.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+packages/*/node_modules
```

Comment:
already ignored due to previous line no ?

Reply:
done